feat(webpack): avoid errors when code is instrumented with harmony module wrapper (#1303)
LavaMoat/lavamoat: packages/webpack/README.md

This plugin will skip policy enforcement for such ignored modules.

#### HMR

> LavaMoat is not compatible with Hot Module Replacement (HMR) and any form of it would not make much sense with the security guarantees, so you're better off disabling LavaMoat for development builds where HMR is required.

Reviewer: ```suggestion
LavaMoat is not compatible with Hot Module Replacement (HMR). Disable LavaMoat for development builds where HMR is required while keeping it enabled for production builds.
```
Brevity, encourage otherwise enabling

This plugin will skip policy enforcement for such ignored modules.

#### HMR

LavaMoat is not compatible with Hot Module Replacement and any form of it would not make much sense with the security guarantees, so you're better off using lavamoat in your production build.

> It's not easy to spot when a tool or setting is enabling HMR though, so LavaMoat plugin will stub out some of the HMR functionality to do nothing.

Reviewer: ```suggestion
LavaMoat is not compatible with Hot Module Replacement (HMR), so you're better off disabling LavaMoat for development builds where this is required. 

The LavaMoat webpack plugin will disable HMR by stubbing out some of the wepack HMR functionality.
```

Author: Sorry, I was wrong, this has nothing to do with HMR, just reuses H next to M :see_no_evil:
